G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailImapCommand.md

```diff
@@ -3,38 +3,46 @@
+| `timeout`      | [timespan](G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutcommand](G1ANT.Language/G1ANT.Addon.Core/Variables/TimeoutCommandVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
+| `errorcall`    | [procedure](G1ANT.Language/G1ANT.Language/Structures/ProcedureStructure.md) | no       |                                                             | Name of a procedure to call when the command throws an exception or when a given `timeout` expires |
+| `errorjump`    | [label](G1ANT.Language/G1ANT.Language/Structures/LabelStructure.md) | no       |                                                             | Name of the label to jump to when the command throws an exception or when a given `timeout` expires |
+| `errormessage` | [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | no       |                                                             | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
+| `errorresult`  | [variable](G1ANT.Language/G1ANT.Language/Structures/VariableStructure.md) | no       |                                                             | Name of a variable that will store the returned exception. The variable will be of [error](G1ANT.Language/G1ANT.Language/Structures/ErrorStructure.md) structure  |
+
+For more information about `if`, `timeout`, `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](G1ANT.Manual/appendices/common-arguments.md) page.
 
 ### Example
 
 ```G1ANT
-♥dateformat = dd\mm\yyyy
-♥yesterday = ⟦date⟧13.01.2018
-mail.imap host imap.gmail.com port 993 login mail@gmail.com password p@$$w0rD sincedate ♥yesterday todate ♥date onlyunreadmessages true markallmessagesasread false result ♥list 
+♥yesterday = ⟦date:dd.MM.yyyy⟧13.01.2019
+mail.imap imap.gmail.com login mail@gmail.com password p@$$w0rD sincedate ♥yesterday todate ♥date onlyunreadmessages true markallmessagesasread false ignorecertificateerrors true result ♥list 
 
 foreach ♥element in ♥list
   dialog ♥element
-end foreach
+end
 ```
 
-In the example above the `mail.imap` command will check GMail IMAP server for unread messages received between January 13, 2018 and the present moment. The matching emails will be displayed in the subsequent dialog boxes (only message subjects are shown, because it’s a default behavior of the mail structure).
+In the example above the `mail.imap` command will check GMail IMAP server for unread messages in the default INBOX folder, received between January 13, 2019 and the present moment. The matching emails will be displayed in the subsequent dialog boxes (only message subjects are shown, because it’s a default behavior of the mail structure).
```

Comment:
```suggestion
In the example above the `mail.imap` command will check Gmail IMAP server for unread messages in the default INBOX folder, received between January 13, 2019 and the present moment. The matching emails will be displayed in the subsequent dialog boxes (only message subjects are shown, because it’s a default behavior of the mail structure).
```